tools/common/SdkVersions.cs.in

```diff
@@ -27,10 +27,10 @@ namespace Xamarin {
 		public const string MinWatchOSCompanionSimulator = "@MIN_WATCHOS_COMPANION_SIMULATOR_VERSION@";
 		public const string MinTVOSSimulator = "@MIN_TVOS_SIMULATOR_VERSION@";
 
-		public const string MaxiOSSimulator = "@MAX_IOS_VERSION@";
-		public const string MaxWatchOSSimulator = "@MAX_WATCH_VERSION@";
-		public const string MaxWatchOSCompanionSimulator = "@MAX_IOS_VERSION@";
-		public const string MaxTVOSSimulator = "@MAX_TVOS_VERSION@";
+		public const string MaxiOSSimulator = "@MAX_IOS_SIMULATOR_VERSION@";
+		public const string MaxWatchOSSimulator = "@MAX_WATCH_SIMULATOR_VERSION@";
+		public const string MaxWatchOSCompanionSimulator = "@IOS_SDK_VERSION@";
+		public const string MaxTVOSSimulator = "@MAX_TVOS_SIMULATOR_VERSION@";
```

Comment:
You'll have to add entries here for the MAX_*_DEPLOYMENT_TARGET variables, and check those in mtouch instead of the current sdk version (where we want to check for the deployment target in mtouch, there might be places where checking the sdk is the right thing to do).